web/src/rest.ts

```diff
@@ -278,12 +295,12 @@ client.interceptors.request.use((config) => ({
   },
 }));
 
-const user = () => dandiRest.user;
```

Comment:
Note, this function is redundant now that we're using composition API. I made the necessary updates to the rest of the codebase to account for the fact that this is a `Ref` instead of a function.